lib/commands/build.js

```diff
@@ -0,0 +1,206 @@
+'use strict'
+
+const RSVP = require('rsvp')
+
+const fs = require('fs-extra')
+const glob = require('globby')
+const npmi = require('npmi')
+const path = require('path')
+
+const Command = require('ember-electron/lib/commands/-command')
```

Comment:
@anulman @felixrieseberg am I missing something here? Using this kind of absolute require breaks for me locally, and is also causing the travis tests to fail. Why aren't all of these relative paths?